dialogue-apache-hc5-client/src/main/java/com/palantir/dialogue/hc5/ScheduledIdleConnectionEvictor.java

```diff
@@ -54,21 +53,16 @@
                             EXECUTOR_NAME)),
                     EXECUTOR_NAME));
 
-    static ScheduledFuture<?> schedule(
-            ConnPoolControl<?> connectionManager, Duration delayBetweenChecks, Duration maxIdleTime) {
-        return schedule(connectionManager, delayBetweenChecks, maxIdleTime, sharedScheduler.get());
+    static ScheduledFuture<?> schedule(ConnPoolControl<?> connectionManager, Duration delayBetweenChecks) {
+        return schedule(connectionManager, delayBetweenChecks, sharedScheduler.get());
     }
 
     private static ScheduledFuture<?> schedule(
-            ConnPoolControl<?> connectionManager,
-            Duration delayBetweenChecks,
-            Duration maxIdleTime,
-            ScheduledExecutorService scheduler) {
+            ConnPoolControl<?> connectionManager, Duration delayBetweenChecks, ScheduledExecutorService scheduler) {
         return scheduler.scheduleWithFixedDelay(
                 () -> {
                     try {
                         connectionManager.closeExpired();
-                        connectionManager.closeIdle(TimeValue.ofMilliseconds(maxIdleTime.toMillis()));
```

Comment:
This acts as an upper bound, our connections already use the value provided by the server, or our default (50 seconds) if the server doesn't send `Keep-Alive: timeout=`